### ISSUE TITLE
YOLO strange point in implementation!

### ISSUE THREAD

**Issue by aif2017** [YunYang1994/TensorFlow2.0-Examples#81]
hello, thank you for your nice job.
in yolo compute_loss function you have used two input conv and pred.
pred used for xywh and conv for conv_raw_prob and conv_raw_conf. 
the only difference between pred and conv in confidence and prob is just applying a sigmoid.
and you used sigmoid_cross_entropy_with_logits..... so there isn't any preference to use conv or pred....
